sites/all/modules/_custom/ajax_calls/ajax_calls.module

```diff
@@ -0,0 +1,73 @@
+<?php
+
+function check_folder_duplicate($folder_id){
+
+	$ret_dat = array('is_duplicate' => NULL, 'exception' => NULL);
+		
+	try {
+		$row_count = db_select('folders')->condition('folder_id', $folder_id)->countQuery()->execute()->fetchField();
+		if($row_count>0){
```

Comment:
shouldn't a folder uniqueness be defined based on the section. Is the section always be same. You may want to update it considering we are giving abilities to make folders for sections. For now I won't ask you to change it but in future my suggestion would be to add a user or a section to this so that if someone else makes it other people can still make the folder. We can discuss this further wth @jonwetzel on this.